docs/meetings/vc-4-apr-2019-summary.md

```diff
@@ -24,7 +24,7 @@ workflow.
 
 **ACTION: Martin to put WIP code on GitHub and make it available to others** - DONE
 
-Ascyncio in WS: not done yet, but not on the critical path (can be done later).
+Asyncio in WS: not done yet, but not on the critical path (can be done later).
```

Comment:
These meetings notes are internal only, but thought it was also harmless to update since the IDE did all the hard work to locate/change the text.